app/Console/Commands/CloseFundoFloraFundingProgrammesCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\V2\FundingProgramme;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Log;
+
+class CloseFundoFloraFundingProgrammesCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'close-fundo-flora-funding-programmes';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Close Fundo Flora funding programmes by setting their status to disabled';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $this->info('Starting to close Fundo Flora funding programmes...');
+
+        // Fundo Flora funding programme UUIDs
+        $fundoFloraUuids = [
+            'e1bdd11c-aefe-4424-ad57-0e8c5ea1e2ab', // Non-profit funding programme
+            '22cdd788-abab-4542-ab6f-d8999aacab9d', // Enterprise funding programme
```

Comment:
Did you get these programme UUIDs from staging or prod? They might not be the same (although we've updated prod to staging recently enough that they _probably_ are).